appinfo/routes.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-return ['routes' => [
```

Comment:
Ah indeed we announce them in the readme https://github.com/nextcloud/groupfolders/#api

Reply:
We can (and should) change the README to use the OCS endpoint URLs, but I agree with @rullzer that we should also keep the old routes for compatibility and to avoid breaking users workflows. @juliushaertl Can you update the PR?

Reply:
Hm, actually the README doesn't mention the base URL, so people actually expect to already be able to use the OCS URLs (`/ocs/v1.php/cloud/`), see #1019. Should I propose a fix to the README that tells people to use `/apps/groupfolders/...`for now, or can we just get this PR merged soon to fix the issue?